source/actions/PetAnalysisRunAction.cc

```diff
@@ -15,17 +15,22 @@
 
 #include <G4Run.hh>
 #include <G4AnalysisManager.hh>
+#include <G4GenericMessenger.hh>
 
 REGISTER_CLASS(PetAnalysisRunAction, G4UserRunAction)
 
-PetAnalysisRunAction::PetAnalysisRunAction(): G4UserRunAction()
+PetAnalysisRunAction::PetAnalysisRunAction(): G4UserRunAction(),
+                                              fname_("OptTrackInfo"),
+                                              ntuple_fname_("OpticalEvtInfo")
 {
+  msg_ = new G4GenericMessenger(this, "/Actions/PetAnalysisRunAction/");
+  msg_->DeclareProperty("histo_file", fname_,
+                        "Name of the file for the analysis histograms");
+  msg_->DeclareProperty("ntuple_file", ntuple_fname_,
+                        "Name of the file for the analysis histograms");
```

Comment:
It has the same description as fname. In this case are all the files of fname?

Reply:
Fixed.